### DISCUSSION TITLE
[Bug?]: 这撅括弧，多逝一件霉事啊

### DISCUSSION THREAD

**Discussion by Abjust** [daizihan233/KuoHuBit#20] [General]
### 请确保您已阅读以上注意事项，并勾选下方的确认框。

- [X] 我已经仔细阅读上述教程和 ["《提问的智慧》"](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)。
- [X] 我确定这不是因为操作失误而引起的问题。
- [X] 我确定这是一个有意义的Issue，且目前没有解决方案。

### 重现步骤

1、撅括弧

### 期望的结果是什么？

撅得动

### 实际的结果是什么？

撅不动

### 补充说明（可选）

别当真（

### 来点色图/Memes

任何色图终将绳之以法，和蔼！

**Comment by Abjust** [comment]
- 你有很多仙贝撅不下吗？
- 喝茶要潇洒一点。
- 撅一个人未必一定要撅他一辈子的。
- 我喜欢一朵花未必一定要把它爆满山。
- 我喜欢homo难道叫他停下来你让我闻一闻？
- 我喜欢张嘉文难道叫他飘下来来撅我？
- 有时候太照顾别人的感受会失去林檎的。
- 轻轻的我走了，正如我轻轻的来。我挥一挥衣袖，不带走一对林檎。

**Comment by daizihan233** [comment]
乙烯一克一克一克，哼哼哼，啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊！！！

**Comment by daizihan233** [comment]
# 你听过林檎吗？转给你身边的所有人！
你知道林檎吗？林檎最近为什么这么火？说到林檎大家肯定都不陌生，但是你知道林檎到底是什么吗？相信大家对林檎都很熟悉，林檎就是我们每天都会经常遇到的，但是林檎是怎么回事呢？林檎其实就是林檎，目前在网上拥有非常高的热度。大家可能会感到很惊讶，林檎为什么是这样的？林檎究竟为什么火起来了呢？但事实就是这样，小编也感到非常惊讶。那么以上就是今天小编为大家整理的关于林檎是什么梗，林檎为什么火的原因。如果大家喜欢，可以点赞表示对小编的支持。欢迎在下方评论和小编一起讨论喔
